tfexec/terraform.go

```diff
@@ -61,7 +61,7 @@ type TerraformCLI interface {
 	Version(ctx context.Context) (string, error)
 
 	// Init initializes the current work directory.
-	Init(ctx context.Context, opts ...string) error
+	Init(ctx context.Context, backendConfig []string, opts ...string) error
```

Comment:
Use the existing `opts` argument instead of adding the new argument for Init.

Reply:
Passing backend config options as opts in 0f5711ef5f0e4890406075af38a9a20427bba2fd